azext_edge/edge/providers/check/cloud_connectors.py

```diff
@@ -246,18 +245,18 @@ def _display_connector_runtime_health(
             ),
         )
         padding += PADDING_SIZE
-        pod_name_prefixes = [
-            f"{prefix}{connector['metadata']['name']}" for connector in connectors
-        ]
-        for pod in pod_name_prefixes:
-            evaluate_pod_health(
-                check_manager=check_manager,
-                target=target,
-                namespace=namespace,
-                pod=pod,
-                display_padding=padding,
-                detail_level=detail_level,
-            )
+        # pod_name_prefixes = [
```

Comment:
commented out but not deleted?

Reply:
for this file Ryan mentioned the logic might be reusable for dataflow logic so he would want to keep this for longer, therefore I thought comment it out so he would still see the structure

Reply:
I think we can probably safely remove the cloud_connector code. Dataflows are different enough that I don't think we'll get much use out of this.

Reply:
got it, will update!